### ISSUE TITLE
after update getting z4h-compile errors

### ISSUE THREAD

**Issue by ciscohack** [romkatv/zsh4humans#320]
after the upgrade getting the below error. request to fix it

-z4h-compile:2249: unmatched '
-z4h-compile:zcompile:35: can't read file: /Users/praverai/.cache/zsh4humans/v5/cache/zcompdump-501-5.9

**Comment by ciscohack** [comment]
@romkatv please do help with this error and one more minor help how can I sort fzf output to show the latest file/directory at bottom when I press shift + down arrow

**Comment by romkatv** [comment]
I've no clue why that error might trigger.

Not sure if the second question has a good answer but probably not. You'll have to read the source code to see if there are any knobs in there.

**Comment by ciscohack** [comment]
@romkatv this error started after i ran z4h update ..so this mean you can't suggest a possible fix for this

**Comment by Sneethe** [comment]
@ciscohack Is this error still occurring?
Whats the output of `cat $Z4H/zsh4humans/version`

**Comment by ciscohack** [comment]
@Sneethe No, I am not facing this error anymore.. what i did i sanitized my zshrc file ..first removed all the config i added on top of default config then re-added one by one to find offended config thats how solved my problem
